common/file_watcher_win.h

```diff
@@ -77,9 +77,14 @@ class FileWatcherWin {
   // Stops watching directory changes.
   absl::Status StopWatching() ABSL_LOCKS_EXCLUDED(modified_files_mutex_);
 
-  // Indicates whether a directory is currently watched.
+  // Indicates whether StartWatching() was called, but StopWatching() was not
+  // called yet.
   bool IsWatching() const;
 
+  // Indicates whether a directory is actively watched for changes. In contrast
+  // to Iswatching(), returns false while the directory does not exist.
+  bool IsRunning() const;
```

Comment:
Nit: I don't think that `IsRunning` is not the best description of what this does, because it reminds me more of a thread running. Maybe `IsActive` would be better, but I don't feel strongly about it.

Reply:
Yeah not happy with the naming, either. "IsWatching" would be ideal since it really means it's watching that directory. I think I'll rename IsWatching to IsStarted and IsRunning to IsWatching.